### PR TITLE
feat: throw error on dividing by zero

### PR DESCRIPTION
**Pull request by Deep-Codes**
Follow up to #83 

Throw an exception `Bhai Bhai Bhai Bhai Bhai ye kya kar diya...` instead of `infinity` 

Can be changed to anything let me know what the text can be.

Screenshot 2022-03-13 at 6 12 58 PM

### CODE REVIEW

**Review by chauhanprateek89 — COMMENTED** [review]
This can be a good starting point to serve custom messages for runtime exceptions. We can add more messages for other exceptions as part of feature enhancements.

**Review by AmiyaDas — APPROVED** [review]
Sahi hai bhai

File: packages/interpreter/src/helpers/index.ts [hunk 1]
```diff
@@ -62,6 +62,10 @@ export function getOperationValue(
 
     case "/=":
     case "/":
+      if (operands.right === 0) {
+        throw new RuntimeException(`Bhai Bhai Bhai Bhai Bhai ye kya kar diya...`)
```
Comment (aniketsingh0104) [inline packages/interpreter/src/helpers/index.ts]: can we change this error message to => `Bhai Bhai Bhai ye kya kar diya...zero se divide ni karte`?

5 bhai's seems, little too much. Also we should tell the developer what is he doing wrong (zero division).

**Review by aniketsingh0104 — CHANGES_REQUESTED** [review]

File: packages/interpreter/src/helpers/index.ts (hunk 1 above)
Comment (Deep-Codes) [reply packages/interpreter/src/helpers/index.ts]: Sure that makes sense!

**Comment by ghost** [comment]
Can you squash the commits into a single commit, so we can avoid unnecessary commits and maintain bettee code history

**Comment by Deep-Codes** [comment]
> Can you squash the commits into a single commit, so we can avoid unnecessary commits and maintain bettee code history

There's an option to squash and merge which should be done while merging.

**Review by aniketsingh0104 — APPROVED** [review]

**Pull request merged**